### PR TITLE
Add option to store passphrase in localstorage or sessionstorage

### PR DESCRIPTION
**Pull request by thomasmarr**
This PR leans heavily on code written by @epicfaace, referenced in issue #120. Original gist [here](https://gist.github.com/epicfaace/c1a4452401af14d35b60fe211f2c1559).

An option has been added to the CLI to permit the user to select between localStorage, sessionStorage, or None.

### CODE REVIEW

**Comment by thomasmarr** [comment]
@robinmoisson would appreciate it if you could take a look at this PR, thanks. If you don't plan to merge please let me know.

File: cli/index.js [hunk 1]
```diff
@@ -43,13 +43,24 @@ const namedArgs = Yargs
           describe: 'Path to custom HTML template with password prompt.',
           default: path.join(__dirname, 'password_template.html')
       })
+      .option('s', {
+          alias: 'storepassphrase',
+          type: 'string',
+          describe: 'Store the passphrase (in clear text) so tht the user doesn\'t have to re-enter on each page load. Valid options are; SessionStorage, LocalStorage, None',
```
Comment (epicfaace) [inline cli/index.js:49]:
```suggestion
          describe: 'Store the passphrase (in clear text) so tht the user doesn\'t have to re-enter on each page load. Valid options are: SessionStorage, LocalStorage, None',
```

**Review by epicfaace — COMMENTED** [review]
This is great -- thanks for doing this!

**Comment by robinmoisson** [comment]
Hey @thomasmarr thanks a lot for PR, it's helpful! And so sorry for taking so long to answer.

I read it and the related discussion, the approach that makes the more sense to me is adding a "Remember me" checkbox with optional expiration so that's what I implemented in #124 (feel free to comment there if you'd like). So I'm closing this one, thanks again for opening it!

**Pull request closed without merging**